creator-node/src/index.js

```diff
@@ -17,13 +17,23 @@ const exitWithError = (...msg) => {
   process.exit(1)
 }
 
-const runDBMigrations = async () => {
+const verifyDBConnection = async () => {
+  logger.info('Verifying DB connection')
+  await sequelize.authenticate() // runs SELECT 1+1 AS result to check db connection
+}
```

Comment:
lets add a log here to say connected successfully?

Reply:
+1 but i think the log should be after line 27 below for consistency, along with the `verifying db connection`. either way we should standardize

Reply:
makes sense - take a look at how I refactored. It made sense to me for the `try/catch` and `exitWithError` for each of `verifyDBConnection` and `runDBMigrations` to live in those functions. I didn't want to import `exitWithError` and `logger` into `migrationManager.js` so there's a few layers of wrapping functions here, but I think overall it makes the code a bit easier to read. 

YMMV so LMK if you prefer that I do it the other way!

Reply:
i'm good with how you did it!